src/Akka.Persistence.Sql/Snapshot/ByteArraySnapshotDao.cs

```diff
@@ -48,6 +107,181 @@ public class ByteArraySnapshotDao : ISnapshotDao
+                                                Manifest = r.Manifest,
+                                                Payload = r.Payload,
+                                                SerializerId = r.SerializerId,
+                                            }).ToListAsync();
+                                    return new SnapshotReadGroup(a, set, err: (Exception?)null);
+                                }
+                                catch (Exception ex)
+                                {
+                                    return new (a, null, err: ex);
+                                }
+                            }
+                            else
+                            {
+                                try
+                                {
+                                    var set = await connection.GetTable<LongSnapshotRow>()
+                                        .Where(r => r.PersistenceId.In(a.Entries.Keys))
+                                        .Select(
+                                            r => new
+                                            {
+                                                Created = r.Created,
+                                                PersistenceId = r.PersistenceId,
+                                                SequenceNumber = r.SequenceNumber,
+                                                Manifest = r.Manifest,
+                                                Payload = r.Payload,
+                                                SerializerId = r.SerializerId,
+                                                RowNum = LinqToDB.Sql.Ext.Rank().Over().PartitionBy(r.PersistenceId).OrderByDesc(r.SequenceNumber).ToValue()
+                                            })
+                                        .Where(r => r.RowNum == 1)
+                                        .Select(
+                                            r => new LongSnapshotRow()
+                                            {
+                                                Created = r.Created,
+                                                PersistenceId = r.PersistenceId,
+                                                SequenceNumber = r.SequenceNumber,
+                                                Manifest = r.Manifest,
+                                                Payload = r.Payload,
+                                                SerializerId = r.SerializerId,
+                                            }).ToListAsync();
+                                    return new (a, set, err: (Exception?)null);
+                                }
+                                catch (Exception ex)
+                                {
+                                    return new (a, null, err: ex);
+                                }
+                            }
+                        }
+                    }).SelectAsync(1,
```

Comment:
IDK if this should -really- be `SelectAsync`, alas I don't have a clean way to do `.Async()` on a Subflow ;_; ???

In general the _desire_ here is to ensure that we can fire off results to things without blocking the next read. Probably a cleaner/better way to do this, open to suggestions. <3